### PR TITLE
Update boskos deployment with mason

### PR DESCRIPTION
**Pull request by krzyzacy**
Also bumped kubetest since there's a client change..
Will try to make another deployment to make sure world is not on fire.

/area boskos

/assign @BenTheElder @sebastienvas @cjwagner 
/hold

### CODE REVIEW

**Comment by BenTheElder** [comment]
discussed this offline, @krzyzacy is testing boskos usage out in a different deployment.

File: boskos/resources.json [hunk 1]
```diff
@@ -1,533 +0,0 @@
-[
```
[the remark this answers is not in the document]
Comment (krzyzacy) [reply boskos/resources.json]: done

**Comment by krzyzacy** [comment]
(ok this should be fine now, internal boskos is also WAI, let's merge tomorrow morning I guess)

**Pull request merged**